tests/testthat/test-s3-methods.R

```diff
@@ -24,28 +24,55 @@ test_that("tidy, summary, and print work", {
 
   ## lm_robust
   lmo <- lm_robust(y ~ x, data = dat, se_type = "classical")
+  lmfo <- lm_robust(y ~ z, data = dat, fixed_effects = ~ x)
 
   capture_output(
     summary(lmo)
   )
 
+  capture_output(
+    summary(lmfo)
+  )
+
+  expect_true(
```

Comment:
`testthat::expect_output`

Reply:
cat \neq output?

Reply:
Github also says that I dismissed your stale reviews. I have done no such thing.